components/toast/index.css

```diff
@@ -90,32 +90,26 @@ governing permissions and limitations under the License.
     background-color: var(--highcontrast-toast-negative-background-color-default, var(--mod-toast-negative-background-color-default, var(--spectrum-toast-negative-background-color-default)));
     color: var(--highcontrast-toast-negative-background-color-default, var(--mod-toast-negative-background-color-default, var(--spectrum-toast-negative-background-color-default)));
 
-    .spectrum-Toast-closeButton {
-      &:focus-visible:not(:active) {
-        color: var(--highcontrast-toast-negative-background-color-default, var(--mod-toast-negative-background-color-default, var(--spectrum-toast-negative-background-color-default)));
-      }
+    .spectrum-Toast-closeButton &:focus-visible:not(:active) {
+      color: var(--highcontrast-toast-negative-background-color-default, var(--mod-toast-negative-background-color-default, var(--spectrum-toast-negative-background-color-default)));
     }
   }
 
   &--info {
     background-color: var(--highcontrast-toast-informative-background-color-default, var(--mod-toast-informative-background-color-default, var(--spectrum-toast-informative-background-color-default)));
     color: var(--highcontrast-toast-informative-background-color-default, var(--mod-toast-informative-background-color-default, var(--spectrum-toast-informative-background-color-default)));
 
-    .spectrum-Toast-closeButton {
-      &:focus-visible:not(:active) {
-        color: var(--highcontrast-toast-informative-background-color-default, var(--mod-toast-informative-background-color-default, var(--spectrum-toast-informative-background-color-default)));
-      }
+    .spectrum-Toast-closeButton &:focus-visible:not(:active) {
+      color: var(--highcontrast-toast-informative-background-color-default, var(--mod-toast-informative-background-color-default, var(--spectrum-toast-informative-background-color-default)));
     }
   }
 
   &--positive {
     background-color: var(--highcontrast-toast-positive-background-color-default, var(--mod-toast-positive-background-color-default, var(--spectrum-toast-positive-background-color-default)));
     color: var(--highcontrast-toast-positive-background-color-default, var(--mod-toast-positive-background-color-default, var(--spectrum-toast-positive-background-color-default)));
 
-    .spectrum-Toast-closeButton {
-      &:focus-visible:not(:active) {
-        color: var(--highcontrast-toast-positive-background-color-default, var(--mod-toast-positive-background-color-default, var(--spectrum-toast-positive-background-color-default)));
-      }
+    .spectrum-Toast-closeButton &:focus-visible:not(:active) {
```

Comment:
I think this selector might be incorrect with the `&` in there.

Reply:
Oh good catch!! I'll double check and see if I can get those fixed